guides/line-management/probation.md

```diff
@@ -0,0 +1,87 @@
+# Probation at Made Tech
+
+When someone joins Made Tech, they're "on probation" for a short time.
+This means their notice period is shorter - usually one week.
+
+At the end of probation, people either pass or fail. Most people pass.
+Rarely, you'll extend probation, or pass someone early.
+
+This guidance is written for line managers, but may be useful for people on probation too.
+
+- [What happens during probation?](#what-happens-during-probation)
+- [Explaining how to pass probation](#explaining-how-to-pass-probation)
+- [One-to-one meetings during probation](#one-to-one-meetings-during-probation)
+- [Finishing probation](#finishing-probation)
+
+## What happens during probation?
+
+In the first week, explain what probation is, and how to pass it.
+
+Each week, meet your direct one-to-one.
+Get feedback from their team, and reflect on it together.
+Talk often about whether they're on-track to pass.
+
+At the end of probation, pass or fail it, and explain what happens next.
+
+## Explaining how to pass probation
+
+Talk about what good looks like for their role. [The handbook role pages](../../roles/README.md) may help with this.
+It's good to spend the first two weeks talking about this.
+
+Explain that feedback is useful evidence on how they're performing.
+
+Tell your direct to ask their team for feedback. You can talk to your direct's team to help get more feedback, if needed.
```

Comment:
"direct" reads a bit weird. "direct report" would be more conventional terminology.

Reply:
agree

Reply:
addressed in https://github.com/madetech/handbook/pull/404/commits/945a407ebf65a885e2429b01ed72126f2f5f6c58